bin/moodle-docker-compose

```diff
@@ -24,8 +24,15 @@ done
 basedir="$( cd -P "$( dirname "$SOURCE" )/../" && pwd )"
 export ASSETDIR="${basedir}/assets"
 
+dockercompose="docker-compose"
 
-dockercompose="docker-compose -f ${basedir}/base.yml"
+# Allow multiple instances run simultaneously
```

Comment:
Hi @jpahullo,
you should use `COMPOSE_PROJECT_NAME` to achieve the same goal w/o changing the code base but eventually the `README.md` to describe how to use it, more or less the statements you've written here.

HTH,
Matteo

Reply:
Hi!

I just discovered this variable thanks to you. The whole spec is here: https://docs.docker.com/compose/reference/envvars/

Just to tell you: My idea was to be independent our variable name from the docker-compose API, that may change over time.

However, I got it:

1. I will remove changes on the bin/moodle-docker-compose.
1. I will update the README.md to inform how to run multiple instances using this `COMPOSE_PROJECT_NAME`.

Regards,

Jordi